contrib/systemd/containerized/origin-master.service

```diff
@@ -1,7 +1,9 @@
 [Unit]
 After=docker.service
 Before=origin-node.service
+PartOf=docker.service
```

Comment:
When systemd stops or restarts docker.service it will also stop or restart origin-master.service.